metrics-report-generator/src/main/scala/it/pagopa/interop/metricsreportgenerator/util/Error.scala

```diff
@@ -0,0 +1,9 @@
+package it.pagopa.interop.metricsreportgenerator.util
+
+import java.util.UUID
+
+object Error {
+  final case class TenantNotFound(tenantId: UUID) extends Exception(s"Tenant ${tenantId.toString} not found")
+  final case class DescriptorNotAllowed(descriptorId: UUID)
+      extends Exception(s"Descriptor ${descriptorId.toString} without activatedAt field")
```

Comment:
if the only reason of this error is the lack of `activatedAt` I would rename this class to `MissingActivationTimestamp`